sdk/communication/Azure.Communication.Chat/src/Models/CommunicationIdentifierModel.cs

```diff
@@ -1,12 +1,22 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+// This is a temporary solution.
+// Please remove this code when swagger is updated to support `kind` property: https://github.com/Azure/azure-rest-api-specs/pull/19675
```

Comment:
perhaps we should create an ADO for it, otherwise it will be easily forgotten

Reply:
@AikoBB I think that when code is regenerated for a newer swagger version, there will be a conflict with this file and it won't compile, so once updating thew swagger and recompiling, a team will already know that something is wrong here and the comment explains why the file was created and why it could be separately removed. ADO for this one could be "update swagger and regenerate your modality code" but I think it's something that is planned by teams anyway, isn't it?

Reply:
@AikoBB  done